feature-extensions/multiaccount/core.py

```diff
@@ -0,0 +1,368 @@
+import boto3
```

Comment:
General question: What's stopping us from doing most of this in Terraform instead of reinventing the wheel in Python?

Reply:
Yeah, There are couple of things which I had to share why we choose boto3 instead of terraform.
1. We dont need to store the secondary accounts credentials in aws credentials/Profiles. Also we have to store the credentials in Jenkins anyway. So the tear down or uninstall will use those credentials.

2. Terraform destroy will destroy all the resources. But now we can add/remove "n" number of accounts, anytime. ie, first Account A with region A has been added. Then later they can add extra regions in the same account or they can remove the existing regions etc. 

So that kind of flexibility is there in boto3, but in terraform, if we need a new region or account we need to update the provider or statefile for the later added accounts. 

Also, provider with alias has limitations, explained here also the issue is still open-  https://github.com/hashicorp/terraform/issues/18682

3. Profile was one another way to use it with terraform, but there also, we need to specify the profile name with regions in terraform provider before runtime.

So altogether, now we have the flexibility to handle the multi account with any regions. I would say we can got to terraform modular approach in future, but need a bit investigation on that side too.